buildSrc/src/main/groovy/com/thoughtworks/go/build/docker/BuildDockerImageTask.groovy

```diff
@@ -111,15 +111,6 @@ class BuildDockerImageTask extends DefaultTask {
         verifyHelper.call(isNativeVerify)
         logger.lifecycle("\nVerification of ${imageNameWithTag} image on ${distro.dockerVerifyArchitecture} successful.")
       }
-
-      logger.lifecycle("Cleaning up...")
-      // delete the image, to save space
-      if (!project.hasProperty('dockerBuildKeepImages')) {
```

Comment:
Why did you need to remove this?

Reply:
Because it was deleting the image that was being built. Even after passing a self check.